documentation/kernel-semantics.tex

```diff
@@ -2167,33 +2167,42 @@ \subsubsection{Super Property Assignment}
     \end{eqfigure}
 \end{figure}
 
+
 \begin{figure}[Htp]
     \begin{eqfigure}
     \begin{align}
     &\begin{multlined}
             \cesktranswheresplit*%
             {\contconf{\InstanceMethodA}{\val{s}}}%
             {\execconf{\stmt}{\env}{[]}{[]}{\strace}{\handler}{\cex,\,\cstrace}{\econt}{\scont}}%
-            {\parbox{10cm}{where $\membermeta = class(\val).lookup(\idmeta)$ is an instance method with body $\stmt$ and formals $\formals$\\
-            $\scont = \ExitSK{\nnull}$\\
-            $\env = \ext{\env_{empty}}{\this :: \formals}{\val :: \val{s}}$}}
+            {\begin{aligned}
+                \text{where } \membermeta &= class(\val).lookup(\idmeta)\text{ is an instance method with body $\stmt$ and formals $\formals$},\\
```

Comment:
I'm ok with it as a temporary solution, but in the final version we need `lookup` to be formally defined.  And it should probably be a function, not a "member".  I don't think we have meta-members :)